examples/entry.rs

```diff
@@ -27,6 +27,7 @@ impl FactoryComponent for Counter {
     type CommandOutput = ();
     type ParentInput = AppMsg;
     type ParentWidget = gtk::Box;
+    type Index = DynamicIndex;
```

Comment:
Should the macro default this to `DynamicIndex`?

Reply:
Good suggestion. Is implemented now.